crypto/src/flow.rs

```diff
@@ -0,0 +1,33 @@
+use penumbra_proto::{dex as pb, Protobuf};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Serialize, Deserialize)]
+#[serde(try_from = "pb::MockFlowCiphertext", into = "pb::MockFlowCiphertext")]
+// TODO: should not be a raw u64, needs to be constant-length
+pub struct MockFlowCiphertext(u64);
+
+// Fake implementation for now, TODO: replace w/ additively homomorphic encryption impl
+impl MockFlowCiphertext {
+    pub fn mock_decrypt(&self) -> u64 {
+        self.0
+    }
+
+    pub fn add(&mut self, amount: u64) {
+        self.0 += amount
+    }
```

Comment:
It'd be more ergonomic to have this be an `impl std::ops::Add`, but not essential to do right now